### PR TITLE
Random change

### PR DESCRIPTION
**Pull request by 1798643961**
Move some code for better handle
Handle worldevent
fix cast error

### CODE REVIEW

**Comment by wdog5** [comment]
image

**Comment by Romaindu35** [comment]
I have run again the workflows, I think that forge maven repository was down when latest workflows was execute !

File: patches/minecraft/net/minecraft/entity/Entity.java.patch [hunk 1]
```diff
@@ -169,20 +169,6 @@
 +   public void inactiveTick() { }
 +   // Spigot end
 +
-+   @Deprecated
-+   public void setBukkitEntity(CraftEntity bukkitEntity) {
```
Comment (Romaindu35) [inline patches/minecraft/net/minecraft/entity/Entity.java.patch:173]: This function is always delete 🙃

**Review by Romaindu35 — CHANGES_REQUESTED** [review]

File: patches/minecraft/net/minecraft/entity/Entity.java.patch (hunk 1 above)
Comment (1798643961) [reply patches/minecraft/net/minecraft/entity/Entity.java.patch:173]: 。。。。。。
I moved the code in Entity.class
Don't worry

Comment (Romaindu35) [reply patches/minecraft/net/minecraft/entity/Entity.java.patch:173]: Ok

**Review by Romaindu35 — APPROVED** [review]

**Pull request merged**